.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parser": "babel-eslint",
+  "extends": "standard",
+  "globals": {
+    "atom": false,
+    "afterEach": false,
+    "beforeEach": false,
+    "describe": false,
+    "fdescribe": false,
+    "xdescribe": false,
+    "expect": false,
+    "it": false,
+    "fit": false,
+    "xit": false,
```

Comment:
Yeah I've seen them committing `fdescribe` once, but I think that's bad practice.